app/__init__.py

```diff
@@ -48,6 +50,7 @@ def create_app(app_name=None):
     aws_ses_client.init_app(application.config['AWS_REGION'])
     notify_celery.init_app(application)
     encryption.init_app(application)
+    clients.init_app(sms_clients=[firetext_client, mmg_client], email_clients=[aws_ses_client])
```

Comment:
We store all the clients in an object, built at start up. Clients in the database MUST be in the object, mappings are between 'identifier' in the database and 'name' in the client object.